llvm/test/tools/sycl-post-link/composite-spec-constant.ll

```diff
@@ -69,6 +57,11 @@ entry:
   ret void
 }
 
+; CHECK: %[[#N0:]] = call i32 @_Z20__spirv_SpecConstantii(i32 [[#ID + 6]], i32
```

Comment:
My idea was to actually move the checks right below the `__sycl_get*` intrinsics which lead to the checked IR.